### PR TITLE
Fix CROTA meta keyword EUVI

### LINKED ISSUE #4845: Fix CROTA in EUVI map source
e.g. 
```python
from sunpy.map import Map

smap = Map('https://stereo-ssc.nascom.nasa.gov/pub/beacon/ahead/secchi/img/euvi/20210107/20210107_001615_n7euA.fts')
print(smap.wcs)
```
raises
```
WARNING: FITSFixedWarning: CROTA = -6.39331135705 
keyword looks very much like CROTAn but isn't. [astropy.wcs.wcs]
```
It would be good to fix this in the EUVI map source.

### PR DESCRIPTION
**Pull request by hayesla**
Fixes #4845

### CODE REVIEW

File: sunpy/map/sources/stereo.py [hunk 1]
```diff
@@ -42,6 +42,10 @@ def __init__(self, data, header, **kwargs):
         # date FITS keyword
         if ('date_obs' in self.meta) and not('date-obs' in self.meta):
             self.meta['date-obs'] = self.meta['date_obs']
+        # fix CROTA to CROTAn
+        if "crota" in self.meta:
```
Comment (dstansby) [inline sunpy/map/sources/stereo.py:46]:
```suggestion
        if "crota" in self.meta and "crota2" not in self.meta:
```
Just to be on the safe side I think it's worth checking there isn't already a crota2.

File: sunpy/map/sources/stereo.py [hunk 2]
```diff
@@ -42,6 +42,10 @@ def __init__(self, data, header, **kwargs):
         # date FITS keyword
         if ('date_obs' in self.meta) and not('date-obs' in self.meta):
             self.meta['date-obs'] = self.meta['date_obs']
+        # fix CROTA to CROTAn
+        if "crota" in self.meta and "crota2" not in self.meta:
+            crota2 = self.meta.pop("crota")
+            self.meta["crota2"] = crota2
```
Comment (Cadair) [inline sunpy/map/sources/stereo.py:48]: Just cause?
```suggestion
            self.meta["crota2"] = self.meta.pop("crota")
```

**Review by Cadair — APPROVED** [review]

**Comment by ayshih** [comment]
Please add a `log.debug()` message noting that this keyword was removed (see the draft SEP sunpy/sunpy-SEP#50).

**Comment by dstansby** [comment]
I've removed the backport to 2.0 since this is a behaviour change.

**Review by dstansby — APPROVED** [review]

**Pull request merged**